plugins/modules/fabric_virtual_networks_workflow_manager.py

```diff
@@ -0,0 +1,2950 @@
+            "multiple_ip_to_mac_addresses": "isMultipleIpToMacAddresses",
+            "supplicant_based_extended_node_onboarding": "isSupplicantBasedExtendedNodeOnboarding",
+            "group_policy_enforcement_enabled": "isGroupBasedPolicyEnforcementEnabled"
+        }
+
+        if vn_name == "INFRA_VN":
+            params_to_remove = ["is_critical_pool", "layer2_flooding_enabled", "fabric_enabled_wireless", "security_group_name",
+                                "ip_directed_broadcast", "intra_subnet_routing_enabled", "multiple_ip_to_mac_addresses"]
+            for item in params_to_remove:
+                gateway_mapping.pop(item, None)
+
+        return gateway_mapping
+
+    def create_anycast_payload(self, anycast, fabric_id):
+        """
+        Constructs the payload for creating an Anycast Gateway configuration.
+        Args:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+            anycast (dict): A dictionary containing Anycast Gateway configuration details.
+            fabric_id (str): The identifier for the fabric associated with the Anycast Gateway.
+        Returns:
+            dict: A dictionary representing the Anycast Gateway payload, structured with the necessary
+                parameters for API submission. This includes:
+                - fabricId: The ID of the fabric.
+                - virtualNetworkName: The name of the virtual network.
+                - ipPoolName: The name of the IP pool.
+                - trafficType: The type of traffic.
+                - Additional parameters mapped from the Anycast configuration, with defaults set as needed.
+        Description:
+            This function creates a payload for the Anycast Gateway API based on the provided configuration
+            details. It retrieves a mapping of parameters needed for the Anycast configuration and populates
+            the payload with values from the `anycast` dictionary.
+            This structured payload then be sent to the API to create/update Anycast Gateway configuration.
+        """
+
+        vn_name = anycast.get("vn_name")
+        anycast_payload = {
+            "fabricId": fabric_id,
+            "virtualNetworkName": vn_name,
+            "ipPoolName": anycast.get("ip_pool_name"),
+            "trafficType": anycast.get("traffic_type"),
+        }
+        anycast_mapping = self.get_anycast_gateway_mapping(vn_name)
+
+        if vn_name == "INFRA_VN":
+            infra_enable_list = ["supplicant_based_extended_node_onboarding", "group_policy_enforcement_enabled"]
+
+            for key, value in anycast_mapping.items():
+                playbook_param = anycast.get(key)
+                if key == "pool_type":
+                    anycast_payload[value] = anycast.get(key, "EXTENDED_NODE")
+                    continue
+
+                if playbook_param is not None:
+                    anycast_payload[value] = playbook_param
```

Comment:
self.log("Adding parameter '{0}' with value '{1}' to payload.".format(key, playbook_param), "DEBUG")